designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/Text.kt

```diff
@@ -123,3 +126,58 @@ fun Description(
         textAlign = TextAlign.Start,
     )
 }
+
+@Composable
+fun ListCardTitle(
```

Comment:
should this be internal?

Reply:
yes it should I'm touching up some things so I'll add this in